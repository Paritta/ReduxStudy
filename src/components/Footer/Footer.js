import React from "react";
import styled from "styled-components";
import oc from "open-color";

const propTypes = {
};

const defaultTypes = {
};

const Wrapper = styled.div`
    display: flex;
    flex-direction: row;
    justify-content: space-around;
    align-items: center;
    
    background: ${oc.gray[2]};
    
    height: 50vh;
`;

const Item = styled.div`
    width: 300px;
    min-width: 200px;
`;

export const Footer = () => {
    return (
        <Wrapper>
            <Item>
                <div>
                    Github:
                </div>
                <div>
                    Email
                </div>
                <div>
                    P.h
                </div>
            </Item>
            <Item>
                <div>
                    P.h
                </div>
            </Item>
        </Wrapper>
    )
};

Footer.propTypes = propTypes;
Footer.defaultTypes = defaultTypes;

export default Footer;
