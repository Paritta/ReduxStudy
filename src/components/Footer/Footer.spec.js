import React from "react";
import { Footer } from "./Footer";
import "jest-styled-components";
import ReactTestRenderer from "react-test-renderer";

it("Card render correctly", () => {
    const tree = ReactTestRenderer.create(
        <Footer />
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
