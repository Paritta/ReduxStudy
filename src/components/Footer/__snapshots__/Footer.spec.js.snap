// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #e9ecef;
  height: 50vh;
}

.c1 {
  width: 300px;
  min-width: 200px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam cumque deleniti eius, incidunt labore laudantium maiores modi reiciendis sequi totam! Facere, quidem quod. A accusantium asperiores, illum libero voluptas voluptate?
    </div>
  </div>
  <div
    className="c1"
  >
    <div>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam cumque deleniti eius, incidunt labore laudantium maiores modi reiciendis sequi totam! Facere, quidem quod. A accusantium asperiores, illum libero voluptas voluptate?
    </div>
  </div>
</div>
`;
